airflow/config_templates/default_celery.py

```diff
@@ -26,7 +26,7 @@
 
 
 def _broker_supports_visibility_timeout(url):
-    return url.startswith("redis://") or url.startswith("sqs://")
+    return url.startswith("redis://") or url.startswith("rediss://") or url.startswith("sqs://")
```

Comment:
```suggestion
    return url.startswith(("redis://", "rediss://", "sqs://"))
```
You can pass a tuple of strings in `.startswith()`. This will then return True if `url` starts with _any_ of those string values.

Reply:
Great option!